### PR TITLE
refactor: improving the details page

### PR DESCRIPTION
**Pull request by sachaudh**
### Description

This PR changes the details page by doing the following:
1. Adding the instance details information
2. Adding a link to the Getting Started Guide
3. Adding a link to the Product Configuration 

Screen Shot 2022-08-26 at 1 12 56 PM
Screen Shot 2022-08-26 at 1 12 58 PM

Mocks: https://www.sketch.com/s/caf22e03-37ae-459f-a32d-f10e3f9c728b/a/My245Y3

### CODE REVIEW

File: src/components/InstanceDetailsList.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Region</DescriptionListTerm>
+        <DescriptionListDescription>
+          {regionValueToLabel(instance.region)}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Sensor Host URL</DescriptionListTerm>
+        <DescriptionListDescription>-</DescriptionListDescription>
```
Comment (dvail) [inline src/components/InstanceDetailsList.js:53]: Is the sensor host URL not defined on the instance object? (TBH I didn't even know sensor was accessible via URL.)

**Review by dvail — APPROVED** [review]
Only small nit here is that the order of the DescriptionList items differs from the one in the mocks, not sure if that was intentional or not though.

File: src/components/InstanceDetailsList.js (hunk 1 above)
Comment (sachaudh) [reply src/components/InstanceDetailsList.js:53]: So based on the wording by the backend team we have two URLs:

1. The Central UI URL
2. The Central URL for Sensor

Hopefully, that makes more sense. Seems like they've settled on the following field names:

1. `centralUIURL`
2. `centralDataURL`

I'll use these names in the UI for now and once their changes come in it'll show the correct data

Comment (sachaudh) [reply src/components/InstanceDetailsList.js]: I changed the order to be the same as the mock as well

**Pull request merged**